lld/ELF/LinkerScript.h

```diff
@@ -228,6 +234,10 @@ class InputSectionDescription : public SectionCommand {
   // SectionPatterns can be filtered with the INPUT_SECTION_FLAGS command.
   uint64_t withFlags;
   uint64_t withoutFlags;
+
+  // If present, input section matching uses class membership instead of file
```

Comment:
If non-empty, this indicates a CLASS(class_name) reference, and filePat/sectionPatterns are not used.

Suggest to move this after sectionPatterns to make it clear they are mutual exclusive.